pkgs/development/haskell-modules/patches/alex-3.2.4-ghc8.8.x.patch

```diff
@@ -0,0 +1,27 @@
+From deaae6eddef5186bfd0e42e2c3ced39e26afa4d6 Mon Sep 17 00:00:00 2001
```

Comment:
Have you created this patch or is it sourced from somewhere?

Reply:
I created it by taking the diff from 3.2.4 up to now which contains fixes for ghc8.8.x, but has not been released yet.. idk how/where to submit a patch like this to, so some help would be nice 😅

Reply:
I think you can fetch patches like this from github though I am not sure

Reply:
Well, it's pointing directly to the commit now, which feels a lot cleaner. Trying to build my other project locally now with this latest commit

Reply:
Nope, doesnt work:
```
Preprocessing executable 'alex' for alex-3.2.4..
Setup: The program 'alex' is required but it could not be found
```
Will have to continue with this PR tomorrow.

Reply:
@luc-tielen Maybe you'd be able to just pull in this single patch:

https://github.com/simonmar/alex/pull/143/commits/deaae6eddef5186bfd0e42e2c3ced39e26afa4d6

```nix
pkgs.fetchpatch {
  url = "https://github.com/simonmar/alex/commit/deaae6eddef5186bfd0e42e2c3ced39e26afa4d6.patch";
  sha256 = "0x1v45rkmj194c41d1nqi3ihj9z4rsy8zvpfcd8p960g1fia7hhh";
})
```

(You'll need to change the sha256.)

Reply:
Yep, that seems to work. Nifty!